openmc/data/njoy.py

```diff
@@ -215,11 +213,21 @@ def make_pendf(filename, pendf='pendf', error=0.001, stdout=False):
              heatr=False, purr=False, acer=False, stdout=stdout)
 
 
-def make_ace(filename, temperatures=None, ace='ace', xsdir='xsdir', pendf=None,
-             error=0.001, broadr=True, heatr=True, gaspr=True, purr=True,
-             acer=True, evaluation=None, **kwargs):
+def make_ace(filename, temperatures=None, ace=None, xsdir=None,
+             output_dir=None, pendf=False, error=0.001, broadr=True,
+             heatr=True, gaspr=True, purr=True, acer=True, evaluation=None,
```

Comment:
Seems like the `acer` and `ace` arguments ought to be combined in the style of the others?